addons/web_editor/static/src/js/widgets/ace.js

```diff
@@ -820,7 +821,14 @@ var ViewEditor = Widget.extend({
      * @private
      */
     _onResChange: function () {
-        this._displayResource(this._getSelectedResource());
+        var selectedResource = this._getSelectedResource().toString();
+        var customFiles = ['/website/static/src/scss/bootstrap_variables_override.scss', '/website/static/src/scss/css_custom_rules.scss', '/website/static/src/scss/css_custom_variables.scss'];
+        if (!customFiles.includes(selectedResource) && selectedResource.endsWith('.scss')) {
+            Dialog.confirm(this, _t("If you edit and save this file, you will never receive updates anymore unless you reset this file."), {
```

Comment:
An alert is too violent, just put a warning message somewhere, but not something that the user must discard to continue.